core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/stringfunction/factory/Tokens.kt

```diff
@@ -20,9 +20,7 @@ data class CustomTokenOptions(var name: Boolean = false, private val specials: B
         queryString = "{\"name\": $name, \"case\": \"$case\", \"specials\": $specials}"
     }
 
-    fun toDopeQuery(): DopeQuery = DopeQuery(queryString, emptyMap())
+    fun toDopeQuery() = DopeQuery(queryString, emptyMap())
 }
 
-fun tokens(inStr: List<String>): TokensExpression = TokensExpression(inStr)
-
-fun tokens(inStr: List<String>, opt: CustomTokenOptions): TokensExpression = TokensExpression(inStr, opt)
+fun tokens(inStr: List<String>, opt: CustomTokenOptions = CustomTokenOptions()) = TokensExpression(inStr, opt)
```

Comment:
Another default parameter added here, intended?

Reply:
The Class `TokensExpression` has this default value, and I think the Function should be as similar to the Class as Possible.